praetorian_cli/ui/aegis/commands/job_helpers.py

```
@@ -335,6 +329,106 @@ def configure_parameters(menu, capability_info, has_credential=False):
+        assets, _ = menu.sdk.assets.list(key_prefix=f"#asset#{hostname}", pages=1)
+
+        # Build choice list
+        choices = []
+        if assets:
+            for asset in assets[:15]:
+                key = asset.get('key', '')
+                name = asset.get('name', '') or key.split('#')[-1]
+                dns = asset.get('dns', '') or (key.split('#')[2] if len(key.split('#')) > 2 else '')
+                choices.append({'key': key, 'name': name, 'dns': dns})
+
+        menu.console.print(f"\n  Select target asset:")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)